sky/provision/kubernetes/instance.py

```diff
@@ -253,6 +253,31 @@ def _run_command_on_pods(node_name, node_namespace, command):
         _request_timeout=kubernetes.API_TIMEOUT)
     return cmd_output
 
+def _run_command_on_pods_with_logs(node_name, node_namespace, command, command_list):
+    cmd_output = kubernetes.stream()(
+        kubernetes.core_api().connect_get_namespaced_pod_exec,
+        node_name,
+        node_namespace,
+        command=command,
+        stderr=True,
+        stdin=True,
+        stdout=True,
+        tty=False,
+        _preload_content=False,
+        _request_timeout=kubernetes.API_TIMEOUT)
+    while cmd_output.is_open():
+        cmd_output.update(timeout=1)
+        if cmd_output.peek_stderr():
```

Comment:
Can we also print out stdout of the commands run?

Reply:
I'm assuming you mean into the `provision.log` file but also happy to directly `print()`!